pages/index.js

```diff
@@ -12,7 +14,21 @@ const Index = () => {
       >
         Docs
       </Heading>
-      <Row sx={{ mt: [6, 7, 8, 9] }}>
+      <Row sx={{ mt: [4, 5, 6, 7] }}>
+        <Column start={[1, 2, 3, 3]} width={[6, 8, 9, 9]}>
+          <Box variant='text.mono'>
+            <Text>
+              You can also check out our{' '}
+              <Link variant='text.mono' href={'tech/data-science'}>
+                data science
+              </Link>{' '}
+              and <Link href={'tech/front-end'}>front-end</Link> contributor
+              guides on how to contribute to our projects.
```

Comment:
I would just remove the frontend docs entirely for now, since it seems like we never actually wrote any content for them.
```suggestion
                data science
              </Link>{' '}
              contributor guide on how to contribute to our projects.
```